addons/gdshell/docs/en/commands/index.md

```diff
@@ -0,0 +1,33 @@
+<a href="https://github.com/Kubulambula/Godot-GDShell">
+  <img src="https://github.com/Kubulambula/Godot-GDShell/blob/main/addons/gdshell/docs/assets/logo.png" align="left" width="80" height="80">
+</a>
+
+# GDShell Command List
```

Comment:
```suggestion
# Default Command List
```

I think that this is a better title for this page as the user would know that they are on the GDShell docs